src/api-docs/source/includes/_wallet_auctions.md

```diff
@@ -1,179 +1,912 @@
 # Wallet - Auctions
 
-Note, all auction related functionality is available through the RPC interface of the handshake wallet. HTTP is not yet supported.
 
+## Get Wallet Names
 
-## getnames (hsw)
+```shell--vars
+id='primary'
+```
+
+```shell--curl
+curl $walleturl/$id/name
+```
 
 ```shell--cli
-hsw-rpc getnames
+> no CLI command available
 ```
 
 ```javascript
-const {NodeClient} = require('hs-client');
+const {WalletClient} = require('hs-client');
 const {Network} = require('hsd');
 const network = Network.get('regtest');
 
-const clientOptions = {
+const walletOptions = {
   network: network.type,
-  port: network.rpcPort,
+  port: network.walletPort,
   apiKey: 'api-key'
 }
 
-const client = new NodeClient(clientOptions);
+const walletClient = new WalletClient(walletOptions);
+const wallet = walletClient.wallet(id);
 
 (async () => {
-  const result = await client.execute('getnames');
+  const result = await wallet.getNames();
   console.log(result);
 })();
 ```
 
-> getnames returns JSON structured like this: 
+> Sample response:
 
 ```json
 [
   {
-    "name": "why",
-    "nameHash": "27b118c11562ebb2b11d94bbc1f23f3d78daea533766d929d39b580a2d37d4a4",
-    "state": "CLOSED",
-    "height": 189,
-    "renewal": 398,
+    "name": "handshake",
+    "nameHash": "3aa2528576f96bd40fcff0bd6b60c44221d73c43b4e42d4b908ed20a93b8d1b6",
+    "state": "BIDDING",
+    "height": 245,
+    "renewal": 245,
     "owner": {
-      "hash": "7bd7c709e5d5e5cc2382f45daad29d280641bf9d1fdf5f88efb6a1809b16a01b",
-      "index": 0
+      "hash": "0000000000000000000000000000000000000000000000000000000000000000",
+      "index": 4294967295
     },
-    "value": 1000000,
-    "highest": 3000000,
-    "data": "00000000",
+    "value": 0,
+    "highest": 0,
+    "data": "",
     "transfer": 0,
     "revoked": 0,
-    "claimed": false,
+    "claimed": 0,
+    "renewals": 0,
+    "registered": false,
+    "expired": false,
     "weak": false,
     "stats": {
-      "renewalPeriodStart": 398,
-      "renewalPeriodEnd": 10398,
-      "blocksUntilExpire": 9917,
-      "daysUntilExpire": 34.43
+      "bidPeriodStart": 251,
+      "bidPeriodEnd": 256,
+      "blocksUntilReveal": 2,
+      "hoursUntilReveal": 0.33
     }
+  }
+]
+```
+
+List the states of all names known to the wallet.
+
+### HTTP Request
+
+`GET /wallet/:id/name`
+
+
+Parameters | Description
+---------- | -----------
+id <br> _string_ | id of wallet you would like to retrieve the name list for
```

Comment:
suggestion: these wallet descriptions are similar to the name descriptions. We could simplify them to "wallet id."